packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -331,14 +365,32 @@ class ClayMultiSelect extends ClayComponent {
 	/**
 	 * @inheritDoc
 	 */
-	attached() {
+	created() {
 		this._itemFocused = null;
+		this._eventHandler = new EventHandler();
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	attached() {
+		this._eventHandler.add(
+			dom.on(document, 'click', this._handleDocClick.bind(this), true)
```

Comment:
Why do we need to listen for `click` on `document`?? Isn't this an input, can't we handle this with simple css? @pat270 ?

Reply:
Hey @jbalsas when the user is focused on the Label and it leaves with the click to another, the focus will remain and another case, it would be when I click directly on the label, the focus should happen.

Reply:
👍